services/vcpkg/vcpkg-version.tester.js

```diff
@@ -0,0 +1,16 @@
+import { isSemver } from '../test-validators.js'
+import { createServiceTester } from '../tester.js'
+
+export const t = await createServiceTester()
+
+t.create('gets the port version of entt')
+  .get('/entt.json')
+  .expectBadge({ label: 'vcpkg', message: isSemver })
```

Comment:
**note**: Not all ports have a semantic version as it depends on the underlying package (see [documentation](https://learn.microsoft.com/en-us/vcpkg/users/versioning#version-schemes)). However, Entt port does in this specific case.